config/boards/orangepi5-plus.wip

```diff
@@ -0,0 +1,35 @@
+# Rockchip RK3588S octa core 4/8/16GB RAM SoC NVMe USB3 USB-C 2.5GbE eMMC
+BOARD_NAME="Orange Pi 5 plus"
+BOARDFAMILY="rockchip-rk3588"
+BOOTCONFIG="orangepi_5_plus_defconfig" # vendor name, not standard, see hook below, set BOOT_SOC below to compensate
+BOOT_SOC="rk3588"
+KERNEL_TARGET="legacy"
+FULL_DESKTOP="yes"
+BOOT_LOGO="desktop"
+BOOT_FDT_FILE="rockchip/rk3588-orangepi-5-plus.dtb"
+BOOT_SCENARIO="spl-blobs"
+WIREGUARD="no"
```

Comment:
?

Reply:
so what's the problem?

Reply:
Why is wireguard disabled?

Reply:
i just copy this line from rock-5b config file. i will test build and run with wireguard on later.

Reply:
i has been tested wireguard can be built and load successful. so i had removed it.